src/OpenRiaServices.Server/Test/OpenRiaServices.Server.Test.csproj

```diff
@@ -26,6 +26,7 @@
   </ItemGroup>
   <ItemGroup>
     <ProjectReference Include="..\..\OpenRiaServices.EntityFramework\Test\DbContextModel\EFDbContextModels.csproj" />
+    <ProjectReference Include="..\..\OpenRiaServices.Server.UnitTesting\Framework\OpenRiaServices.Server.UnitTesting.csproj" />
```

Comment:
What are the intentions of the projects OpenRiaServices.Server.Test and OpenRiaServices.Server.UnitTesting ? Should they be merged?

Reply:
* OpenRiaServices.Server.UnitTesting is the nuget package with unit testing helpers
* OpenRiaServices.Server.Test tests for the OpenRiaServices.Server project